documentation/docs/05-load.md

```diff
@@ -121,7 +123,11 @@ import * as api from '$lib/api';
 
 /** @type {import('./$types').PageLoad} */
 export async function load({ depends }) {
-	depends(`${api.base}/foo`, `${api.base}/bar`);
+	depends(
+		`${api.base}/foo`,
+		`${api.base}/bar`
+		'page:foobar'
```

Comment:
I think something like this probably communicates 'put what you like here' better:

```suggestion
		'my-stuff:foo'
```